ariadne/cardano/src/Ariadne/Wallet/Backend.hs

```diff
@@ -30,7 +30,7 @@ import Ariadne.Wallet.Face
 data WalletPreface = WalletPreface
     { wpBListener :: !BListenerHandle
     , wpAddUserSecret :: !(TVar UserSecret -> IO ())
-    , wpMakeWallet :: !(CardanoFace -> ((Doc -> IO ()) -> WalletFace, IO ()))
+    , wpMakeWallet :: !(CardanoFace -> ((Doc -> IO ()) -> WalletFace, FilePath -> IO ()))
```

Comment:
Meh, it will be unnecessary after #288 because key file path will be in the Wallet configuration.